fontique/src/family.rs

```diff
@@ -22,8 +23,9 @@ impl FamilyId {
     #[allow(clippy::new_without_default)]
     pub fn new() -> Self {
         /// Make sure this is larger than the largest generic family id.
-        static ID_COUNTER: AtomicU64 = AtomicU64::new(64);
-        Self(ID_COUNTER.fetch_add(1, Ordering::Relaxed))
+        static ID_COUNTER: AtomicCounter = AtomicCounter::new(64);
+        #[allow(clippy::useless_conversion, reason = "Not useless on 32-bit platforms")]
+        Self(ID_COUNTER.fetch_add(1, Ordering::Relaxed).into())
```

Comment:
This `fetch_add` being potential 32 bit introduces a new overflow point, and so needs an overflow comment (same in the other instance of this). This can be the same as in Linebender Resource Handle.